source/2_platform/rex_windows/src/win_window_class.cpp

```diff
@@ -15,15 +15,15 @@ namespace rex
       Internal()
           : window_class()
           , name()
-          , hinstance(NULL)
+          , hinstance(nullptr)
           , registered(false)
       {
       }
 
       bool create(HInstance hInstance, WindowProcedureFunc wndProc, const char8* title)
       {
         name      = title;
-        hinstance = (hInstance == nullptr) ? static_cast<HInstance>(GetModuleHandleA(NULL)) : hInstance;
+        hinstance = (hInstance == nullptr) ? static_cast<HInstance>(GetModuleHandleA(nullptr)) : hInstance;
```

Comment:
When using the WIN32 API I rather use NULL instead of nullptr. Although I am not consistent but that is another review comment.

Reply:
clang-tidy complain when using NULL, which is why I changed it